tests/web3js/eth_non_interactive_test.js

```diff
@@ -1,112 +1,132 @@
 
-it('get transaction', async() => {
-    let blockTx = await web3.eth.getTransactionFromBlock(conf.startBlockHeight, 0)
-    assert.isNotNull(blockTx)
-
-    let tx = await web3.eth.getTransaction(blockTx.hash)
-    assert.deepEqual(blockTx, tx)
-    assert.isString(tx.hash)
-    assert.equal(tx.blockNumber, conf.startBlockHeight)
-    assert.isAbove(parseInt(tx.gas), 1)
-    assert.isNotEmpty(tx.from)
-    assert.isNotEmpty(tx.r)
-    assert.isNotEmpty(tx.s)
-    assert.equal(tx.transactionIndex, 0)
-
-    let rcp = await web3.eth.getTransactionReceipt(tx.hash)
-    assert.isNotEmpty(rcp)
-    assert.equal(rcp.blockHash, blockTx.blockHash)
-    assert.equal(rcp.blockNumber, conf.startBlockHeight)
-    assert.equal(rcp.from, tx.from)
-    assert.equal(rcp.to, tx.to)
-    assert.equal(rcp.cumulativeGasUsed, tx.gas) // todo check
-    assert.equal(rcp.transactionHash, tx.hash)
-    assert.equal(rcp.status, conf.successStatus)
-    assert.equal(rcp.gasUsed, tx.gas)
+it('get transaction', async () => {
+  let blockTx = await web3.eth.getTransactionFromBlock(conf.startBlockHeight, 0)
+  assert.isNotNull(blockTx)
+
+  let tx = await web3.eth.getTransaction(blockTx.hash)
+  assert.deepEqual(blockTx, tx)
+  assert.isString(tx.hash)
+  assert.equal(tx.blockNumber, conf.startBlockHeight)
+  assert.isAbove(parseInt(tx.gas), 1)
+  assert.isNotEmpty(tx.from)
+  assert.isNotEmpty(tx.r)
+  assert.isNotEmpty(tx.s)
+  assert.equal(tx.transactionIndex, 0)
+
+  let rcp = await web3.eth.getTransactionReceipt(tx.hash)
+  assert.isNotEmpty(rcp)
+  assert.equal(rcp.blockHash, blockTx.blockHash)
+  assert.equal(rcp.blockNumber, conf.startBlockHeight)
+  assert.equal(rcp.from, tx.from)
+  assert.equal(rcp.to, tx.to)
+  assert.equal(rcp.cumulativeGasUsed, tx.gas) // todo check
+  assert.equal(rcp.transactionHash, tx.hash)
+  assert.equal(rcp.status, conf.successStatus)
+  assert.equal(rcp.gasUsed, tx.gas)
 })
 
-it('get mining status', async() => {
-    let mining = await web3.eth.isMining()
-    assert.isFalse(mining)
+it('get mining status', async () => {
+  let mining = await web3.eth.isMining()
+  assert.isFalse(mining)
 })
 
-it('get syncing status', async() => {
-    let height = await web3.eth.getBlockNumber()
-    assert.equal(height, conf.startBlockHeight)
+it('get syncing status', async () => {
+  let height = await web3.eth.getBlockNumber()
+  assert.equal(height, conf.startBlockHeight)
 
-    let syncInfo = await web3.eth.isSyncing()
-    // conf.startBlockHeight = 3n
-    assert.equal(syncInfo.startingBlock, height)
-    assert.equal(syncInfo.currentBlock, height)
-    assert.equal(syncInfo.highestBlock, height)
+  let syncInfo = await web3.eth.isSyncing()
+  // conf.startBlockHeight = 3n
+  assert.equal(syncInfo.startingBlock, height)
+  assert.equal(syncInfo.currentBlock, height)
+  assert.equal(syncInfo.highestBlock, height)
+})
+
+it('get fee history', async () => {
+  let response = await helpers.callRPCMethod('eth_feeHistory', ['0x10','latest',[20]])
```

Comment:
why not use the web3 native method https://docs.web3js.org/libdocs/Web3Eth#getfeehistory

Reply:
Nice catch, I wasn't able to find this endpoint. Updated in https://github.com/onflow/flow-evm-gateway/pull/253/commits/88b821b27cf44b556edd342bc88a503a1ab721ad